.github/workflows/dh-ci-frontend.yml

```diff
@@ -40,12 +40,11 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Set up Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version-file: ${{ env.NODE_VERSION_FILE }}
-
-      - name: 📥 Monorepo install
-        uses: ./.github/actions/yarn-nm-install
+          cache: yarn
+      - run: yarn install --immutable
```

Comment:
suggestion: let's give this task a `name`. The comment applies to all `yarn install` tasks bellow.

Reply:
Good idea